tracing/src/main/java/io/strimzi/test/tracing/OpenTelemetryHandle.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.tracing;
+
+import java.util.Map;
+import java.util.Properties;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.kafkaclients.KafkaTracing;
+import io.opentelemetry.instrumentation.kafkaclients.TracingConsumerInterceptor;
+import io.opentelemetry.instrumentation.kafkaclients.TracingProducerInterceptor;
+import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.streams.KafkaClientSupplier;
+
+public class OpenTelemetryHandle implements TracingHandle {
+    @Override
+    public String type() {
+        return "OpenTelemetry";
+    }
+
+    @Override
+    public String envName() {
+        return "OTEL_SERVICE_NAME";
+    }
+
+    @Override
+    public String serviceName() {
+        String serviceName = System.getenv(envName());
+        if (serviceName == null) {
+            serviceName = System.getProperty("otel.service.name");
+        } else {
+            System.setProperty("otel.service.name", serviceName);
+        }
+        if (serviceName != null && System.getenv("OTEL_TRACES_EXPORTER") == null) {
+            System.setProperty("otel.traces.exporter", "jaeger");
+        }
```

Comment:
You set the default here as well as in the bash script ... would it be cleaner to do it only in one place?

Reply:
This is just in case there is no run script.
I can remove it.

Reply:
> Well, It didn't had test-client-tracing dependency from the same project before. So I think you can remove it here.

It needs to be defined in one place.
It looked like this was the place before -- for any libs used in sub modules.

Reply:
> This is just in case there is no run script.
I can remove it.

Why don't you remove it from the script? I think is makes sense _ just in case there is no run script_ ... but why have it in the run script then?

Reply:
It's removed from the script.